draft-ietf-quic-recovery.md

```diff
@@ -778,13 +778,7 @@ Pseudocode for SetLossDetectionTimer follows:
 
 ~~~
  SetLossDetectionTimer():
-    // Don't arm timer if there are no ack-eliciting packets
-    // in flight.
-    if (no ack-eliciting packets in flight):
-      loss_detection_timer.cancel()
-      return
-
-    if (crypto packets are in flight):
+    if (crypto packets are in flight || handshaking client):
```

Comment:
I agree the pseudocode doesn't match the text, but I think the term "handshaking client" is not specific enough a definition to use in the pseudocode.

Reply:
Yeah, I wasn't sure what to go with. Should I introduce a new defined variable?